torch/onnx/symbolic_helper.py

```diff
@@ -173,7 +173,7 @@ def _is_value(x):
 
 
 def _is_tensor_list(x):
-    return x.type().isSubtypeOf(ListType.ofTensors())
+    return isinstance(x.type(), torch._C.ListType) and isinstance(x.type().getElementType(), torch._C.TensorType)
```

Comment:
note: this change was added due to that `x.type().isSubtypeOf(ListType.ofTensors())` returns False when x is of type list of tensor with known scalar type.